### ISSUE TITLE
Advertising use of key id instead of user id in docs

### ISSUE THREAD

**Issue by ooq** [grafeas/kritis#387]
**Is your feature request related to a problem? Please describe.**
We advertise the following workflow to generate key-pair using `gpg` (in docs, bug reproduction workflow, etc.):
```
gpg --quick-generate-key --yes my.attestor@example.com
gpg --armor --export my.attestor@example.com > gpg.pub
gpg --armor --export-secret-keys my.attestor@example.com > gpg.priv
```
This can cause reproduction issue because:

- `gpg` by default does substring match on provided email/user id. e.g.,  `attestor@example.com` is also a match for the above commands
- Multiple keys can be created for the same email/user id, thus a user might not exported the key intended.

**Describe the solution you'd like**
Update the doc to use key id instead of email/user id.

**Describe alternatives you've considered**
Or we can ask to first delete all keys under the user/email id that is to be used. A user might not want to delete.

**Additional context**
If possible, we should add instruction on key deletion in `clean up` section of the tutorials, to avoid key accumulation.

**Comment by ooq** [comment]
A more reproducible steps could look like this:

1. ### Generate a new key with `gpg`.
```
gpg --quick-generate-key --yes my.attestor@example.com
```
The output should contain a 16-digit hexadecimal key id:
```console
$ gpg --quick-generate-key --yes my.attestor@example.com
...
gpg: key B8F04E5568F2C21B marked as ultimately trusted
...
```
2.  ### Save the hexadecimal key  to `KEYID`.
```
KEYID=B8F04E5568F2C21B
```
3.  ### Export the key using `KEYID`.
```
gpg --armor --export $KEYID > gpg.pub
gpg --armor --export-secret-keys $KEYID > gpg.priv
```

**Comment by ooq** [comment]
Alternatively:

1. ### Generate a new key with `gpg`.
```
GPG_OUTPUT="$(gpg --quick-generate-key --yes attestor@example.com)"
```
2.  ### Extract the 40-digit fingerprint from output  to `KEY_FINGER_PRINT`.
```
echo $GPG_OUTPUT | sed -n 's/.*\([A-Z0-9]\{40\}\).*/\1/p'
```
3.  ### Export the key using `KEY_FINGER_PRINT`.
```
gpg --armor --export $KEY_FINGER_PRINT > gpg.pub
gpg --armor --export-secret-keys $KEY_FINGER_PRINT > gpg.priv
```

**Comment by ooq** [comment]
Fixed by #388
